### ISSUE TITLE
better styling for activity streams

### ISSUE THREAD

**Issue by cspanring** [SomervilleCC/interactivesomerville#28]
...they look kind of clunky currently...

**Comment by michaelmessina** [comment]
Agreed, but not terrible. What does it look like when you left-align the author post timestamp/byline? It might be easier on the eyes to read. And perhaps shrink the width of the space between the idea and the byline maybe like 9 or 10 pt? Another thought is to make the idea in the activity stream a couple of shades darker than the background if there's an easy way to do that. Just some thoughts... that hopefully wouldn't take too much to test out.

**Comment by cspanring** [comment]
we're having a designer coming in this week, so hopefully we have them look better soon...
